config/environments/development.rb

```diff
@@ -39,8 +39,11 @@
 
   config.action_mailer.default_url_options = { :host => ENV['DOMAIN'] }
   config.action_mailer.asset_host = ENV['DOMAIN']
-  config.action_mailer.perform_deliveries = ENV['SEND_EMAIL_IN_DEVELOPMENT'] == 'true'
```

Comment:
`config.action_mailer.perform_deliveries` is [`true` by default](http://guides.rubyonrails.org/action_mailer_basics.html).

> Determines whether deliveries are actually carried out when the deliver method is invoked on the Mail message. By default they are, but this can be turned off to help functional testing.